# This message is a container for a Turtlebot's MMA-generated name, and its Decawave-generated grid.
string name
nav_msgs/OccupancyGrid map